modin/pandas/test/test_groupby.py

```diff
@@ -1305,11 +1375,26 @@ def test_groupby_multiindex(groupby_kwargs):
         pytest.param(
             {"by": ["item0", "one", "two"]},
             id="col_name+level_name",
+            marks=pytest.mark.skipif(
+                PandasCompatVersion.CURRENT == PandasCompatVersion.PY36,
+                reason="Weird exception in Modin internals",
```

Comment:
Don't we want to file an issue for this?

Reply:
Generally, we would, but I skipped the tests which were not working when a groupby resulted in zero-row dataframe, and I think this is were the line should be drawn.

I mean, I don't see a point of submitting an issue which no one is ever planning to fix.

Reply:
ok, probably the message above makes sense for this too

> wait until users explicitly request that support